examples/similarity_search/add_faces.py

```diff
@@ -1,23 +1,35 @@
 import argparse
 from aperturedb import Utils, ParallelLoader
-from facenet import generate_embedding
 import dbinfo
 from CelebADataKaggle import CelebADataKaggle
+from aperturedb.transformers.facenet_pytorch_embeddings import FacenetPytorchEmbeddings
+from aperturedb.transformers.common_properties import CommonProperties
+from aperturedb.transformers.image_properties import ImageProperties
 
 search_set_name = "similar_celebreties"
 
 
 def main(params):
     utils = Utils.Utils(dbinfo.create_connector())
     utils.remove_descriptorset(search_set_name)
-    utils.add_descriptorset(search_set_name, 512,
-                            metric="L2", engine="FaissFlat")
-
-    dataset = CelebADataKaggle(
-        records_count=params.images_count,
-        embedding_generator=generate_embedding,
-        search_set_name=search_set_name
-    )
+
+    dataset = CelebADataKaggle()
+
+    # Here's a pipeline that adds extra properties to the celebA dataset
+    dataset = CommonProperties(
+        dataset,
+        adb_data_source="celebA",
```

Comment:
```suggestion
        adb_data_source="kaggle-celebA",
```

Reply:
Just want to be more clear where data came from but ok if you want to leave it at celebA